Packs/CommonPlaybooks/Playbooks/playbook-Block_URL_-_Generic_v2.yml

```diff
@@ -1275,5 +1402,6 @@ inputs:
   playbookInputQuery:
 outputs: []
 tests:
-- No tests (auto formatted)
+- Send Investigation Summary Reports - Test
```

Comment:
how this test relevant for the block URL playbook?

Reply:
Good catch - probably the auto-formatting 🤷 Removing those tests